src/Dropdown.elm

```diff
@@ -420,42 +449,43 @@ update (Config config) msg (State state) data =
                                 _ ->
                                     True
 
-                        focusedItem =
+                        maybeFocusedItem =
                             data
                                 |> List.indexedMap (\i item -> ( i, item ))
                                 |> List.filter (\( i, _ ) -> i == state.focusedIndex)
                                 |> List.head
                                 |> Maybe.map Tuple.second
 
-                        ( cmd, newSelectedItem ) =
+                        cmd =
```

Comment:
There is a slight change in behaviour here - now the onSelectMsg receives the full list of items in the case of a multiselect dropdown, which I think is natural.